packages/use-contractkit/src/connectors/connectors.ts

```diff
@@ -144,11 +145,11 @@ export class InjectedConnector implements Connector {
     if (!ethereum) {
       throw new Error('Ethereum wallet not installed');
     }
-    this.type = window.ethereum?.isMetaMask
+    this.type = ethereum.isMetaMask
       ? WalletTypes.MetaMask
       : WalletTypes.Injected;
     const web3 = new Web3(ethereum);
-    await ethereum.enable();
+    void (await ethereum.request({ method: 'eth_requestAccounts' }));
```

Comment:
This is what MetaMask wants us to do instead of `ethereum.enable` now and when I add it the eslint config requires me to add a void since I'm not doing anything with the resulting value.

Reply:
So eslint won't let you make an `await function()` call when function returns something and its unused? That seems strange. We should modify the eslint config in that case